engine/src/main/resources/assets/i18n/menu_id.lang

```diff
@@ -0,0 +1,255 @@
+    "enum-editor-prompt": "Tentukan nilai untuk node:",
+    "enum-editor-title": "Editor Enum Node",
+    "error-downloading-server-list": "Error sangat mengundah daftar server!",
+    "exit-game": "Keluar dari Terasology",
+    "extra-lighting": "Pencahayaan Ekstra",
+    "eye-adaption": "Adaptasi mata",
+    "failed-to-join": "Gagal untuk masuk",
+    "flickering-lights": "Lampu Berkedip",
+    "fog": "Kabut",
+    "fov-amount": "Banyak FoV",
+    "framerate-limit": "Limit framerate",
+    "game-name": "Nama Permainan",
+    "game-settings": "Pengaturan",
+    "game-worlds": "Dunia",
+    "gameplay": "Template Dunia",
+    "gameplay-info": "bisa  meng-klik 'modul' untuk menyesuaikan lebih lanjut.",
+    "host-multiplayer-game": "Host Permainan",
+    "ingame-menu-title": "menu Permainan",
+    "input-settings": "Input",
+    "input-settings-title": "Pengaturan Input",
+    "inspection-set": "Setel",
+    "invert-mouse": "Balikan Kontrol Mouse",
+    "invert-x": "Balikan Sumbu X",
+    "invert-y": "Balikan Sumbu Y",
+    "invert-z": "Balikan Sumbu Y",
+    "join-game-online": "Masuk Permainan",
+    "join-server-refresh": "Refresh",
+    "join-server-action": "Masuk",
+    "join-server-requested": "Diminta",
+    "join-server-title": "Masuk Server",
+    "light-shafts": "Tiang Cahaya",
+    "listed-servers": "Terdaftar",
+    "load-game": "Muat",
+    "menu-animations": "Animasikan Menu",
+    "missing-name-message": "Anda harus memasukan nama",
+    "module-dependencies-empty": "Modul ini tidak memeliki dependensi",
+    "module-dependencies-exist": "Dependensi Modul",
+    "module-filter": "Filter",
+    "module-list": "Modul",
+    "module-name": "Nama",
+    "module-status": "Status",
+    "module-status-activated": "Actif",
+    "module-status-activegameplay": "Gameplay aktif",
+    "module-status-available": "Tersedia",
+    "module-status-dependency": "Dependensi",
+    "module-status-error": "Dependensi tidak kompatibel atau tidak ditemukan",
+    "module-status-notpresent": "Tidak ada",
+    "module-version-available": "Online",
+    "module-version-installed": "Terinstall",
+    "module-version-installed-none": "Tidak terinstall",
+    "motion-blur": "Motion Blur",
+    "mouse-sensitivity": "Sensitifitas Modul",
+    "movement-dead-zone": "Gerakan Sumbu Mati",
+    "music-volume": "Volume Lagu",
+    "new-binding": "Ikatan baru",
+    "next-toolbar-item": "Jatuhkan barang toolbar selanjutnya",
+    "none": "Tidak ada",
+    "not-bound": "tidak terikat",
+    "nui-editor-alternative-locale": "Pratinjau widget lokal",
+    "nui-editor-assign": "menetapkan",
+    "nui-editor-close": "Tutup",
+    "nui-editor-copy": "Copy",
+    "nui-editor-disable-autosave": "Matikan autosave",
+    "nui-editor-disable-icons": "Matikan pandangan pohon icon",
+    "nui-editor-layout": "layout",
+    "nui-editor-new": "baru",
+    "nui-editor-override": "Simpan",
+    "nui-editor-paste": "Paste",
```

Comment:
I think there _should_ be a translation for "Paste" (and "Redo"/"Undo" a little further down). [14]